### DISCUSSION TITLE
mesh network protocol(layer 3) for EdgeMesh

### DISCUSSION THREAD

**Discussion by kuzm1ch** [kubeedge/kubeedge#5249] [Q&A]
(I am new in this technology)
Do I need a mesh network layer protocol like B.A.T.M.A.N to make the edge network work? 
Is there any documentation on which protocol will fit best in such a scenario?

**Comment by ravaga** [comment]
As far as I know, you don't need another mesh network layer protocol, just install KubeEdge and then EdgeMesh to achieve network connectivity among all the edge nodes.
